File: src/test/java/main/BasicTenderNavigation.java
package main;
import org.testng.annotations.BeforeMethod;
import org.testng.annotations.Test;
import static org.testng.Assert.assertTrue;
public class BasicTenderNavigation extends OpenTenderTest {

    private PublicTenderPage PublicTenderPage;
@BeforeMethod
public void setUpTestEnvironment() throws InterruptedException {
}
    @Test
    public void navigationPublicTenderTab() throws InterruptedException {
        TendersPage tendersPage = new TendersPage(driver);
        tendersPage.switchToBrowserFrame();

        PublicTenderPage publicTenderPage = new PublicTenderPage(driver);
        publicTenderPage.publicTenderTab().click();
        assertTrue(publicTenderPage.publicTenderTab().isEnabled());
    }

    @Test(description = "Navigation to First Tab- Tender Information")
    public void basicNavigation() throws InterruptedException {

        TendersPage tendersPage = new TendersPage(driver);
        tendersPage.switchToBrowserFrame();

        main.PublicTenderPage publicTenderPage = new PublicTenderPage(driver);
        publicTenderPage.tenderInformationTab().click();

        assertTrue(publicTenderPage.tenderInformationTab().isEnabled());
    }



    @Test(description = "Navigation through all modules", groups = "navigateModules")
    public void navigateThoughtMainPanels() throws InterruptedException {

        TendersPage tendersPage = new TendersPage(driver);
        tendersPage.switchToBrowserFrame();
        PublicTenderPage publicTenderPage = new PublicTenderPage(driver);
        publicTenderPage.clickOnElement(publicTenderPage.getTenderInformationTab());
        publicTenderPage.clickOnElement(publicTenderPage.getSubmissionRequirementsTab());
        publicTenderPage.clickOnElement(publicTenderPage.getTenderSpecificationDocumentsTab());
        publicTenderPage.clickOnElement(publicTenderPage.getQuestionsAndAnswersTab());
        publicTenderPage.clickOnElement(publicTenderPage.getMessagesfromTenderAdministratorTab());
        publicTenderPage.clickOnElement(publicTenderPage.getUsersTab());
        publicTenderPage.clickOnElement(publicTenderPage.getLogFilesTab());
        publicTenderPage.clickOnElement(publicTenderPage.getGuidanceTab());
        publicTenderPage.clickOnElement(publicTenderPage.getSubmittedTenderOffersTab());
        publicTenderPage.clickOnElement(publicTenderPage.getTenderResultTab());

    }
}

File: src/test/java/main/BasicTenderNavigation.kt
import main.OpenTenderTest
import main.PublicTenderPage
import main.TendersPage
import org.testng.Assert
import org.testng.annotations.BeforeMethod
import org.testng.annotations.Test
import org.openqa.selenium.By
import com.codeborne.selenide.SelenideElement

class BasicTenderNavigation : OpenTenderTest() {
    private lateinit var publicTenderPage: PublicTenderPage

    @BeforeMethod
    fun setUpTest() {
        val tendersPage = TendersPage(driver)
        tendersPage.switchToBrowserFrame()
        publicTenderPage = PublicTenderPage(driver)
    }

    @Test(description = "Navigation Public Tender Tab")
    fun navigationPublicTenderTab() {
        publicTenderPage.publicTenderTab().click()
        Assert.assertTrue(publicTenderPage.publicTenderTab().isEnabled)
    }

    @Test(description = "Navigation to First Tab- Tender Information")
    fun basicNavigation() {
        publicTenderPage.tenderInformationTab?.click()
        publicTenderPage.tenderInformationTab?.let { Assert.assertTrue(it.isEnabled) }
    }

    @Test(description = "Navigation through all modules", groups = ["navigateModules"])
    fun navigateThoughtMainPanels() {
        publicTenderPage.tenderInformationTab?.click()
        publicTenderPage.submissionRequirementsTab?.click()
        publicTenderPage.tenderSpecificationDocumentsTab?.click()
        publicTenderPage.questionsAndAnswersTab?.click()
        publicTenderPage.messagesfromTenderAdministratorTab?.click()
        publicTenderPage.usersTab?.click()
        publicTenderPage.logFilesTab?.click()
        publicTenderPage.guidanceTab?.click()
        publicTenderPage.submittedTenderOffersTab?.click()
        publicTenderPage.tenderResultTab?.click()
    }
}
